test/lib/jdk/test/whitebox/WhiteBox.java

```diff
@@ -556,6 +560,50 @@ public void clearInlineCaches(boolean preserve_static_stubs) {
   // Force Full GC
   public native void fullGC();
 
+  // Infrastructure for waitForReferenceProcessing()
+  private Method waitForReferenceProcessingMethod = null;
+
+  private Method getWaitForReferenceProcessingMethod() {
+    Method wfrp = waitForReferenceProcessingMethod;
+    if (wfrp == null) {
+      try {
+        wfrp = Reference.class.getDeclaredMethod("waitForReferenceProcessing");
+        wfrp.setAccessible(true);
+        assert wfrp.getReturnType() == Boolean.class;
+        assert wfrp.getParameterCount() == 0;
```

Comment:
I don't think this is necessary, as the absence of an array argument to `getDeclaredMethod` should find only methods with zero args.